models.py

```diff
@@ -46,9 +46,13 @@ class user_post(UserMixin, db.Model):
     postTitle = db.Column(db.String(50), nullable=False)
     postLikes = db.Column(db.Integer)
     RestaurantName = db.Column(db.String(120))
+    image = db.image_attachment('post_picture')
     user_id = db.Column(db.Integer, db.ForeignKey("user.id"))
     post_comments = db.relationship("post_comments", backref="user", lazy=True)
 
+class post_picture(UserMixin, db.Model, db.Image):
```

Comment:
Quick question: What is this class used for?

Reply:
used to create and store the image - I'm working through what it actually does through implementation from the docs but it's basically to create the image that is stored